README.md

```diff
@@ -279,7 +289,461 @@ for the value to be credited to your account.
+
+### Query IssuingHolders
+
+You can query multiple holders according to filters.
+
+```c#
+using starkinfra;
+
+IEnumerable<IssuingHolder> issuingHolders = IssuingHolder.Query(
+    after: new DateTime(2019, 1, 1),
+    before: new DateTime(2022, 3, 1)
+);
+foreach(IssuingHolder issuingHolder in issuingHolders)
+{
+    Console.Write(issuingHolder);
+}
+```
+
+### Delete an IssuingHolder
+
+To cancel a single Issuing Holder by its id, run:
+
+```c#
+using starkinfra;
+
+IssuingHolder issuingHolder = IssuingHolder.Delete(id: "5353197895942144");
+
+Console.Write(issuingHolder);
+```
+
+### Get an IssuingHolder
+
+To get a single Issuing Holder by its id, run:
+
+```c#
+using starkinfra;
+
+IssuingHolder issuingHolder = IssuingHolder.Get(id: "5353197895942144");
+
+Console.Write(issuingHolder);
+```
+
+### Query IssuingHolder logs
+
+You can query holder logs to better understand holder life cycles.
+
+```c#
+using starkinfra;
+
+List<IssuingHolder.Log> logs = IssuingHolder.Log.Query(
+    limit: 10
+    ).ToList();
+foreach (IssuingHolder.Log log in logs)
+{
+    Console.Write(log);
+}
+```
+
+### Get an IssuingHolder log
+
+You can also get a specific log by its id.
+
+```c#
+using starkinfra;
+
+IssuingHolder.Log getLog = IssuingHolder.Log.Get("6299741604282368");
+
+Console.Write(getLog);
+```
+
+### Create IssuingCards
+
+You can issue cards with specific spending rules.
+
+```c#
+using starkinfra;
+
+List<IssuingCard> issuingCards = IssuingCard.Create(
+    new List<IssuingCard> {
+        new IssuingCard(
+            holderName : "Developers",
+            holderTaxId : "012.345.678-90",
+            holderExternalId : "672",
+            rules : new List<Dictionary<string, object>> (){
+                new Dictionary<string, object>{
+                    { "name", "general" },
+                    { "interval", "week" },
+                    { "amount", 50000 },
+                    { "currencyCode", "USD" }
+                }
+            }
+        )
+    }
+);
+
+foreach(IssuingCard issuingCard in issuingCards)
+{
+    Console.Write(issuingCard);
+}
+```
+
+### Query IssuingCards
+
+You can get a list of created cards given some filters.
+
+```c#
+using starkinfra;
+
+IEnumerable<IssuingCard> issuingCards = IssuingCard.Query(
+    after: new DateTime(2019, 1, 1),
+    before: new DateTime(2022, 3, 1)
+);
+foreach(IssuingCard issuingCard in issuingCards)
+{
+    Console.Write(issuingCard);
+}
+```
+
+### Get an IssuingCard
+
+After its creation, information on a card may be retrieved by its id.
+
+```c#
+using starkinfra;
+
+IssuingCard issuingCard = IssuingCard.Get("5353197895942144");
+
+Console.Write(issuingCard);
+```
+
+### Update an IssuingCard
+
+You can update a specific card by its id.
+
+```c#
+using starkinfra;
+
+IssuingCard issuingCard = IssuingCard.Update(id: "5353197895942144", status: "blocked");
+
+Console.Write(issuingCard);
+
+```
+
+### Delete an IssuingCard
```

Comment:
Cancel and IssuingCard
Isso vai mudar no python tbm, mas já adianta aqui pra aproveitar o PR